src/plugins/management/public/management_sections_service.ts

```diff
@@ -94,8 +98,9 @@ export class ManagementSectionsService {
 
   start({ capabilities }: SectionsServiceStartDeps) {
     this.getAllSections().forEach((section) => {
-      if (capabilities.management.hasOwnProperty(section.id)) {
-        const sectionCapabilities = capabilities.management[section.id];
+      const capabilityId = MANAGEMENT_ID_TO_CAPABILITIES[section.id] || section.id;
```

Comment:
@ruanyl Just curious to know why don't we instead update the Id in the Capabilities to `opensearch-dashboards` instead of mapping it here?

Reply:
@manasvinibs Good question, that's for compatibility, `opensearch-dashboards` maybe deep linked, for example, in function test repo.

Reply:
@kavilla Is there a way to estimate the impact of changing capabilities key from `opensearchDashboards` to `opensearch-dashboards` in `src/plugins/management/server/capabilities_provider.ts`?

Reply:
I've seen some places in OSD core uses `capabilities.management.opensearchDashboards`, also there might other plugins are already using this capability key, I think changing this key would be a breaking change, so I think it would be better to do key mapping. What do you think? @manasvinibs